<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="test_errorfail_output" errors="1" failures="1" tests="1" time="0.03" hostname="unknown" timestamp="2015-04-14T11:55:44">
  <properties/>
  <testcase classname="test_errorfail_output" name="test_success">
    <error><![CDATA[Error in tear_down fixture:
Undefined function or variable 'unknown_function_call'.
In test_errorfail_output.m at line 13
In tear_down.m at line 21
In run_test.m at line 73
In run_suite.m at line 63
In recursive_test_run.m at line 77
In recursive_test_run.m at line 42]]></error>
    <failure><![CDATA[(no message available)
In test_errorfail_output.m at line 9]]></failure>
  </testcase>
  <system-out/>
  <system-err/>
</testsuite>
